tools/postman-collections/am-lib-testing.postman_collection.json

```diff
@@ -0,0 +1,38 @@
+{
+	"info": {
+		"name": "am-lib-testing",
+		"_postman_id": "76c19111-7d09-a05a-583c-8ec144ee0601",
+		"description": "",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Create Resource Access",
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "Content-Type",
+						"value": "application/json"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "{\n\t\"resourceId\": \"123\",\n\t\"accessorId\": \"SIGNOF\",\n\t\"explicitPermissions\": {\"userPermissions\": [\"CREATE\", \"UPDATE\", \"HIDE\", \"DELETE\"]}\n}"
+				},
+				"url": {
+					"raw": "localhost:2703/create-resource-access",
```

Comment:
Not for this PR

but if you have spare time at the end of this sprint can you please extract "localhost" and port to vars so that we can then reuse it for different envs (and if anything needs to be changed it's much easier to maintain).

Reply:
Yep, will keep it on my todo list, probably as a separate PR though